src/sentry/backup/helpers.py

```diff
@@ -52,15 +52,16 @@ class CryptoKeyVersion(NamedTuple):
 
 
 # TODO(getsentry/team-ospo#215): These should be options, instead of hard coding.
-DEFAULT_CRYPTO_KEY_VERSION = CryptoKeyVersion(
-    project_id=gcp_project_id(),
-    location="global",
-    key_ring="relocation",
-    key="relocation",
-    # TODO(getsentry/team-ospo#190): This version should be pulled from an option, rather than hard
-    # coded.
-    version="1",
-)
+def get_default_crypto_key_version() -> CryptoKeyVersion:
+    return CryptoKeyVersion(
+        project_id=gcp_project_id(),
```

Comment:
should this be cached? it's going to call to the gcp api each time this is invoked

Reply:
Figured this way was nice in that it supported hot reload during dev a bit better, but I suppose that's premature optimization. Cached with size 1.